sqlglot/dialects/tsql.py

```diff
@@ -755,6 +756,7 @@ class Generator(generator.Generator):
             exp.DataType.Type.TIMESTAMP: "DATETIME2",
             exp.DataType.Type.TIMESTAMPTZ: "DATETIMEOFFSET",
             exp.DataType.Type.VARIANT: "SQL_VARIANT",
+            exp.DataType.Type.ROWVERSION: "ROWVERSION",
```

Comment:
shouldn't we just map rowversion -> binary for tsql, and then have binary map to rowversion? we don't need a rowversion data type

Reply:
The docs specify the following:

_A nonnullable rowversion column is semantically equivalent to a binary(8) column. A nullable rowversion column is semantically equivalent to a varbinary(8) column._

Since the semantics are dynamic, I thought that allowing the `ROWVERSION` type would be best (with that in mind, might be safer to generate `Type.ROWVERSION -> VARBINARY` to all other dialects ?)

Reply:
> might be safer to generate `Type.ROWVERSION -> VARBINARY` to all other dialects ?

That makes sense to me.